defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-apache_mirror: http://apache.mirrors.tds.net
+kafka_apache_mirror: http://apache.mirrors.tds.net
```

Comment:
Why the name change? We have a few things that download from an apache mirror and it is nice to set only one variable to update the mirror for all of our roles.